apps/total-typescript/src/pages/[article].tsx

```diff
@@ -40,6 +41,16 @@ export const getStaticProps: GetStaticProps = async ({params}) => {
       },
     }))
 
+  const shortenedArticleBody =
+    articleBody &&
+    (await serializeMDX(take(article.body?.split('\n'), 6).join('\n'), {
+      useShikiTwoslash: true,
+      syntaxHighlighterOptions: {
+        authorization: process.env.SHIKI_AUTH_TOKEN,
+        endpoint: process.env.SHIKI_ENDPOINT,
+      },
+    }))
```

Comment:
because we serializeMDX on the server, it was going to be over complicated to try and shorten the article length on the client side so this adds a new `shortenedArticleBody` prop that will display if `showLoginWall` is true and user is not logged in